cli/tools/lint.rs

```diff
@@ -204,11 +204,11 @@ fn collect_lint_files(files: &FilesConfig) -> Result<Vec<PathBuf>, AnyError> {
 }
 
 pub fn print_rules_list(json: bool, maybe_rules_tags: Option<Vec<String>>) {
-  let lint_rules = get_configured_rules(LintRulesConfig {
-    exclude: None,
-    include: None,
-    tags: maybe_rules_tags,
-  });
+  let lint_rules = if maybe_rules_tags.is_none() {
+    rules::get_all_rules()
+  } else {
+    rules::get_filtered_rules(maybe_rules_tags, None, None)
+  };
```

Comment:
I'm not sure how to handle filtering on rules that have no tags nicely here. Any suggestions?

Reply:
I think the easiest way would be to add something like `--untagged-only`

Reply:
Here is an implementation example of untagged-only https://github.com/denoland/deno/commit/ba76af6dd7fcb5a9d7e37a95751bc10297b3be13

Reply:
I'm not a fan of adding a specialized flag. Maybe we could support empty valuer in `--rules-tags` that would filter rules with no tags?

Reply:
`--rules-tags=""` feels unintuitive to me

Maybe Its better if we leave it as is for now

If you still think it's a good idea I can implement it

Reply:
I don't feel strongly about it. We can leave it for a follow up